gcloud.go

```diff
@@ -32,7 +32,7 @@ func NewGCloudClient(projectId string, zone string) (gcloud *GCloud, err error)
 	client, err := google.DefaultClient(gcloud.Context, compute.ComputeScope)
 
 	if err != nil {
-		err = fmt.Errorf("Error creating compute client: %v", err)
+		err = fmt.Errorf("Error creating compute client:\n%v", err)
```

Comment:
for zerolog you want to replace this with `log.Error().(err).Msg("Error creating compute client")`